[DM-1424]: Changed user guide configuration docs for getting setting config on a device + screenshots. (#625)
SoftwareAG/c8y-docs: content/users-guide/device-management-bundle/managing-device-data.md

#### To retrieve and apply a configuration snapshot

> The managing of configurations from and to a device can be done in multiple ways. Depending on user permissions and settings of the device user may be able to work with text based, typed file-based and legacy file-based configurations. More detailed and technical information can be found in the [Device management library](/reference/device-management-library/#configuration).

Reviewer: ```suggestion
Managing configurations from and to a device can be done in multiple ways. Depending on user permissions and device settings, you can work with text based, typed file-based or legacy file-based configuration. Refer to [Device management library > Configuration](/reference/device-management-library/#configuration) in the *Reference guide* for more detailed and technical information.
```

additional question: what does "configurations from and to a device" mean exactly?

Author: Means sending a configuration to a device and requesting a configuration from a device. Basically managing configurations on devices.

Reviewer: ok, i added that to the suggestion

> #### To retrieve and apply a configuration snapshot to a device which supports typed file-based configurations

Reviewer: ```suggestion
#### To retrieve and apply a configuration snapshot to a device which supports typed file-based configuration
```

> Typed file-based configurations are the recomended way of hadnling configurations on a device. Here devices can manage multiple configurations at the same time. For different types different configurations can be uploaded/retrieved. The file communication is aslo updated with this approach and handled in a more efficient way.

Author: What I mean is that the typed file-based config handles the files through events that are responsible for the clean up later if device is deleted by themselves and the legacy one uses binary upload. But maybe I need to put it more clear I will think of another way to put it.

> Typed file-based configurations are the recomended way of hadnling configurations on a device. Here devices can manage multiple configurations at the same time. For different types different configurations can be uploaded/retrieved. The file communication is aslo updated with this approach and handled in a more efficient way.

Reviewer: ```suggestion
We recommend you to use typed file-based configuration. With typed file-based configuration, devices can manage multiple configurations at the same time. You can upload or retrieve different configurations for different types. The file communication is also updated with this approach and handled in a more efficient way.
```

> We recommend you to use typed file-based configuration. With typed file-based configuration, devices can manage multiple configurations at the same time. You can upload or retrieve different configurations for different types. Using this approach is considered to be more versatile because instead of files the configurations are handled as events, which can give more efficiency.

Reviewer: ```suggestion
We recommend you to use typed file-based configuration. With typed file-based configuration, devices can manage multiple configurations at the same time. You can upload or retrieve different configurations for different types. Using this approach is more versatile because the configurations are handled as events rather than as files, which is more efficient.

Under **Available supported configurations**, only configuration files with a matching configuration type property or without a configuration type defined are displayed. Also, configuration files are filtered based on the device type.
{{< /c8y-admon-info >}}

> #### To retrieve and apply a configuration snapshot to a device which supports legacy file-based configurations

Reviewer: ```suggestion
#### To retrieve and apply a configuration snapshot to a device which supports legacy file-based configuration
```

> Devices that want to manage configuration as files can achieve a basic form using legacy file-based configuration. This way only allows one configuration to be set per a device.

Reviewer: ```suggestion
Devices managing configuration as files can do so in a basic form using legacy file-based configuration. Legacy file-based configuration only allows a single configuration to be set per a device.
```
